plugins/main/bridge/bridge.go

```diff
@@ -122,7 +122,7 @@ func calcGateways(result *current.Result, n *NetConf) (*gwInfo, *gwInfo, error)
 
 		// Add a default route for this family using the current
 		// gateway address if necessary.
-		if n.IsDefaultGW && !gws.defaultRouteFound {
+		if n.IsDefaultGW {
```

Comment:
@mccv1r0 is this to allow multiple default routes for the bridge or something?

Reply:
Yes, but keeping it seems to behave too, so I added `&& !gws.defaultRouteFound` back.